src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Fix issue where `ExplicitBucketHistogramConfiguration` could be used to
```

Comment:
We need to message the wording that -- as of today, changing aggregations are not supported. This PR is making sure that unsupported Views are ignored, falling back to defaults..
Some wording to indicate that a future version can allow changing aggregations...